<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabsWidget">
      <property name="toolTip">
       <string/>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="searchAddonsTab">
       <attribute name="title">
        <string>Install Addons</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="searchAddonsLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="xpacLayout">
          <item>
           <layout class="QHBoxLayout" name="xpacRadioLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QRadioButton" name="searchVanillaButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Vanilla</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QRadioButton" name="searchTbcButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>TBC</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QRadioButton" name="searchWotlkButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Wotlk</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QRadioButton" name="searchCataButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Cata</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLineEdit" name="searchBar"/>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QPushButton" name="searchButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="myAddonsTab">
       <attribute name="title">
        <string>Add Clients</string>
       </attribute>
       <widget class="QPushButton" name="addClientButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>320</y>
          <width>371</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Client</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>130</y>
          <width>301</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QRadioButton" name="addClientCata">
           <property name="text">
            <string>Cata</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="addClientVanilla">
           <property name="text">
            <string>Vanilla</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="addClientTbc">
           <property name="text">
            <string>TBC</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="addClientWotlk">
           <property name="text">
            <string>Wotlk</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="addonListTab">
       <attribute name="title">
        <string>My Addons</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
